Build fails on macOS 15 with latest 1.32.1

It seems failed to build 1.32.1 with macOS 15: https://github.com/Homebrew/homebrew-core/pull/196023/files, any idea why?

some errors:

```
source/extensions/network/dns_resolver/cares/dns_impl.cc:113:25: error: use of undeclared identifier 'ARES_OPT_UDP_MAX_QUERIES'
  113 |     options.optmask_ |= ARES_OPT_UDP_MAX_QUERIES;
      |                         ^
source/extensions/network/dns_resolver/cares/dns_impl.cc:114:22: error: no member named 'udp_max_queries' in 'ares_options'
  114 |     options.options_.udp_max_queries = udp_max_queries_;
      |     ~~~~~~~~~~~~~~~~ ^
2 errors generated.
```

```
source/common/http/http2/codec_impl.cc:1971:3: error: use of undeclared identifier 'nghttp2_option_set_max_continuations'; did you mean 'nghttp2_option_set_max_settings'?
 1971 |   nghttp2_option_set_max_continuations(options_, 512);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |   nghttp2_option_set_max_settings
/usr/local/include/nghttp2/nghttp2.h:2726:21: note: 'nghttp2_option_set_max_settings' declared here
 2726 | NGHTTP2_EXTERN void nghttp2_option_set_max_settings(nghttp2_option *option,
      |                     ^
source/common/http/http2/codec_impl.cc:1999:3: error: use of undeclared identifier 'nghttp2_option_set_max_continuations'; did you mean 'nghttp2_option_set_max_settings'?
 1999 |   nghttp2_option_set_max_continuations(options_, 1024);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |   nghttp2_option_set_max_settings
/usr/local/include/nghttp2/nghttp2.h:2726:21: note: 'nghttp2_option_set_max_settings' declared here
 2726 | NGHTTP2_EXTERN void nghttp2_option_set_max_settings(nghttp2_option *option,
      |                     ^
2 errors generated.
```

unfortunately, I didn't make the lucky.
```console
xcrun --show-sdk-version
15.2

bazel clean --expunge
INFO: Invocation ID: 8db9566d-f739-4f68-8834-62e20ee48cd3
INFO: Starting clean (this may take a while). Consider using --async if the clean takes more than several minutes.

bazel build //:envoy
Starting local Bazel server and connecting to it...
INFO: Invocation ID: 17cb8c35-fd6b-4520-acce-3a9988bad227
DEBUG: /private/var/tmp/_bazel_zirain/76b59558ddc3194746941b691ede49bd/external/com_google_protobuf/protobuf.bzl:654:10: The py_proto_library macro is deprecated and will be removed in the 30.x release. switch to the rule defined by rules_python or the one in bazel/py_proto_library.bzl.
INFO: Analyzed target //:envoy (1113 packages loaded, 43893 targets configured).
INFO: From Linking external/com_google_cel_cpp/eval/public/libunknown_function_result_set.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_cel_cpp/eval/public/libunknown_function_result_set.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From GoLink external/com_envoyproxy_protoc_gen_validate/protoc-gen-validate_/protoc-gen-validate [for tool]:
ld: warning: ignoring duplicate libraries: '-lm'
INFO: From Linking external/com_github_grpc_grpc/src/core/libdirectory_reader.a:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'directory_reader.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/_objs/directory_reader/0/directory_reader.o(directory_reader.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/_objs/directory_reader/1/directory_reader.o(directory_reader.o)'
INFO: From Linking external/com_google_protobuf/src/google/protobuf/io/libio_win32.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_protobuf/src/google/protobuf/io/libio_win32.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/libgpr.a:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'cpu.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/cpu.o(cpu.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/cpu.o(cpu.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'cpu.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/cpu.o(cpu.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/3/cpu.o(cpu.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'cpu.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/3/cpu.o(cpu.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/2/cpu.o(cpu.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'log.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/3/log.o(log.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/2/log.o(log.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'log.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/2/log.o(log.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/log.o(log.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'log.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/log.o(log.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/4/log.o(log.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'log.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/4/log.o(log.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/log.o(log.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'stat.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/stat.o(stat.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/stat.o(stat.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'string.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/string.o(string.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/string.o(string.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'string.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/string.o(string.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/2/string.o(string.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'sync.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/2/sync.o(sync.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/sync.o(sync.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'sync.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/sync.o(sync.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/sync.o(sync.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'thd.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/thd.o(thd.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/thd.o(thd.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'time.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/time.o(time.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/2/time.o(time.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'time.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/2/time.o(time.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/time.o(time.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'tmpfile.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/2/tmpfile.o(tmpfile.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/tmpfile.o(tmpfile.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'tmpfile.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/1/tmpfile.o(tmpfile.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/gpr/0/tmpfile.o(tmpfile.o)'
INFO: From Linking external/com_google_protobuf/src/google/protobuf/io/libio_win32.a [for tool]:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-opt-exec-ST-a828a81199fe/bin/external/com_google_protobuf/src/google/protobuf/io/libio_win32.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/core/libenv.a:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'env.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/_objs/env/0/env.o(env.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/_objs/env/1/env.o(env.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'env.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/_objs/env/1/env.o(env.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/_objs/env/2/env.o(env.o)'
INFO: From Linking external/com_github_grpc_grpc/src/core/libtchar.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/libtchar.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking source/common/common/libperf_tracing_lib.lo:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/source/common/common/libperf_tracing_lib.lo the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/proto/grpc/lookup/v1/librls_proto_descriptor.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/proto/grpc/lookup/v1/librls_proto_descriptor.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/src/google/protobuf/libany_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_protobuf/src/google/protobuf/libany_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/proto/grpc/gcp/libalts_handshaker_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/proto/grpc/gcp/libalts_handshaker_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_googleapis/google/rpc/libstatus_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_googleapis/google/rpc/libstatus_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/clusters/common/dns/v3/libpkg_cc_proto.lo:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/clusters/common/dns/v3/libpkg_cc_proto.lo the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/proto/grpc/health/v1/libhealth_proto_descriptor.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/proto/grpc/health/v1/libhealth_proto_descriptor.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_googleapis/google/api/libhttpbody_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_googleapis/google/api/libhttpbody_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/src/google/protobuf/compiler/libplugin_proto.upb.a [for tool]:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-opt-exec-ST-a828a81199fe/bin/external/com_google_protobuf/src/google/protobuf/compiler/libplugin_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_googleapis/google/api/libhttp_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_googleapis/google/api/libhttp_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/src/google/protobuf/libwrappers_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_protobuf/src/google/protobuf/libwrappers_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/src/google/protobuf/libtimestamp_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_protobuf/src/google/protobuf/libtimestamp_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/src/google/protobuf/libdescriptor_proto.upb.a [for tool]:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-opt-exec-ST-a828a81199fe/bin/external/com_google_protobuf/src/google/protobuf/libdescriptor_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/upb_generator/reflection/protoc-gen-upbdefs [for tool]:
ld: warning: ignoring duplicate libraries: '-lm'
INFO: From Linking external/com_google_protobuf/src/google/protobuf/libstruct_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_protobuf/src/google/protobuf/libstruct_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/src/google/protobuf/libempty_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_protobuf/src/google/protobuf/libempty_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/src/google/protobuf/libdescriptor_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_protobuf/src/google/protobuf/libdescriptor_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_googleapis/google/api/libannotations_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_googleapis/google/api/libannotations_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_protobuf/src/google/protobuf/libduration_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_protobuf/src/google/protobuf/libduration_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/proto/grpc/lb/v1/libload_balancer_proto_descriptor.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/proto/grpc/lb/v1/libload_balancer_proto_descriptor.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/proto/grpc/lookup/v1/librls_config_proto_descriptor.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/proto/grpc/lookup/v1/librls_config_proto_descriptor.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_googleapis/google/api/expr/v1alpha1/libsyntax_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_googleapis/google/api/expr/v1alpha1/libsyntax_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_google_googleapis/google/api/expr/v1alpha1/libchecked_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_google_googleapis/google/api/expr/v1alpha1/libchecked_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/dev_cel/proto/cel/expr/libsyntax_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/dev_cel/proto/cel/expr/libsyntax_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/core/libwindows_native_resolver.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/libwindows_native_resolver.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/dev_cel/proto/cel/expr/libchecked_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/dev_cel/proto/cel/expr/libchecked_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/core/libwindows_iocp.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/libwindows_iocp.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_envoyproxy_protoc_gen_validate/validate/libvalidate_proto.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_envoyproxy_protoc_gen_validate/validate/libvalidate_proto.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/core/libwindows_endpoint.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/libwindows_endpoint.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/core/libwindows_event_engine_listener.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/libwindows_event_engine_listener.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/annotations/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/annotations/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/core/libwindows_event_engine.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/src/core/libwindows_event_engine.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_cncf_xds/xds/annotations/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_cncf_xds/xds/annotations/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_cncf_xds/xds/type/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_cncf_xds/xds/type/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_cncf_xds/xds/core/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_cncf_xds/xds/core/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_cncf_xds/xds/type/matcher/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_cncf_xds/xds/type/matcher/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_cncf_xds/udpa/annotations/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_cncf_xds/udpa/annotations/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/clusters/aggregate/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/clusters/aggregate/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/load_balancing_policies/client_side_weighted_round_robin/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/load_balancing_policies/client_side_weighted_round_robin/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/load_balancing_policies/pick_first/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/load_balancing_policies/pick_first/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/type/http/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/type/http/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/http/stateful_session/cookie/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/http/stateful_session/cookie/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/type/metadata/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/type/metadata/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/type/tracing/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/type/tracing/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/type/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/type/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/filters/common/fault/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/filters/common/fault/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/overload/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/overload/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/type/matcher/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/type/matcher/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/libiomgr.a:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'endpoint.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/iomgr/1/endpoint.o(endpoint.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/iomgr/0/endpoint.o(endpoint.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'tcp_client.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/iomgr/1/tcp_client.o(tcp_client.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/com_github_grpc_grpc/_objs/iomgr/0/tcp_client.o(tcp_client.o)'
INFO: From Linking external/envoy_api/envoy/config/core/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/core/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/filters/http/stateful_session/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/filters/http/stateful_session/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/load_balancing_policies/common/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/load_balancing_policies/common/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/load_balancing_policies/ring_hash/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/load_balancing_policies/ring_hash/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/data/accesslog/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/data/accesslog/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/metrics/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/metrics/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/trace/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/trace/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/transport_sockets/tls/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/transport_sockets/tls/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/route/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/route/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/rbac/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/rbac/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/filters/http/rbac/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/filters/http/rbac/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/filters/http/fault/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/filters/http/fault/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/accesslog/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/accesslog/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/filters/network/http_connection_manager/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/filters/network/http_connection_manager/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/filters/http/router/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/filters/http/router/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/upstreams/http/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/upstreams/http/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/listener/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/listener/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/common/matcher/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/common/matcher/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/tap/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/tap/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/endpoint/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/endpoint/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/cluster/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/cluster/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/extensions/load_balancing_policies/wrr_locality/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/extensions/load_balancing_policies/wrr_locality/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/config/bootstrap/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/config/bootstrap/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_grpc_grpc/src/compiler/grpc_cpp_plugin [for tool]:
ld: warning: ignoring duplicate libraries: '-lm', '-lpthread'
INFO: From Linking external/envoy_api/envoy/admin/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/admin/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/service/status/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/service/status/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/service/load_stats/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/service/load_stats/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/envoy_api/envoy/service/discovery/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/envoy_api/envoy/service/discovery/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_cncf_xds/xds/data/orca/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_cncf_xds/xds/data/orca/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/com_github_cncf_xds/xds/service/orca/v3/libpkg.upb.a:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/external/com_github_cncf_xds/xds/service/orca/v3/libpkg.upb.a the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking source/extensions/load_balancing_policies/round_robin/libround_robin_lb_lib.lo:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/source/extensions/load_balancing_policies/round_robin/libround_robin_lb_lib.lo the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking source/extensions/transport_sockets/tcp_stats/libtcp_stats_lib.lo:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/source/extensions/transport_sockets/tcp_stats/libtcp_stats_lib.lo the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking source/common/matcher/libmatcher_lib.lo:
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: bazel-out/darwin_arm64-fastbuild/bin/source/common/matcher/libmatcher_lib.lo the table of contents is empty (no object file members in the library define global symbols)
INFO: From Linking external/v8/libv8_libshared_noicu.lo:
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'allocation.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/1/allocation.o(allocation.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/0/allocation.o(allocation.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'factory.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/1/factory.o(factory.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/0/factory.o(factory.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'free-list.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/1/free-list.o(free-list.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/0/free-list.o(free-list.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'graph-visualizer.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/1/graph-visualizer.o(graph-visualizer.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/0/graph-visualizer.o(graph-visualizer.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'graph.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/1/graph.o(graph.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/0/graph.o(graph.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'heap.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/0/heap.o(heap.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/1/heap.o(heap.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'objects-printer.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/1/objects-printer.o(objects-printer.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/0/objects-printer.o(objects-printer.o)'
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: warning duplicate member name 'sweeper.o' from 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/1/sweeper.o(sweeper.o)' and 'bazel-out/darwin_arm64-fastbuild/bin/external/v8/_objs/v8_libshared_noicu/0/sweeper.o(sweeper.o)'
ERROR: /Users/zirain/Codes/servicemesh/envoy/source/extensions/network/dns_resolver/cares/BUILD:11:19: Compiling source/extensions/network/dns_resolver/cares/dns_impl.cc failed: (Exit 1): cc_wrapper.sh failed: error executing CppCompile command (from target //source/extensions/network/dns_resolver/cares:config) external/local_config_cc/cc_wrapper.sh -U_FORTIFY_SOURCE -fstack-protector -Wall -Wthread-safety -Wself-assign -Wunused-but-set-parameter -Wno-free-nonheap-object -fcolor-diagnostics ... (remaining 208 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
source/extensions/network/dns_resolver/cares/dns_impl.cc:113:25: error: use of undeclared identifier 'ARES_OPT_UDP_MAX_QUERIES'
  113 |     options.optmask_ |= ARES_OPT_UDP_MAX_QUERIES;
      |                         ^
source/extensions/network/dns_resolver/cares/dns_impl.cc:114:22: error: no member named 'udp_max_queries' in 'ares_options'
  114 |     options.options_.udp_max_queries = udp_max_queries_;
      |     ~~~~~~~~~~~~~~~~ ^
2 errors generated.
Target //source/exe:envoy-static failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 103.273s, Critical Path: 13.74s
INFO: 20003 processes: 10273 remote cache hit, 9714 internal, 15 darwin-sandbox, 1 local.
ERROR: Build did NOT complete successfully
```

--- a/bazel/foreign_cc/BUILD
+++ b/bazel/foreign_cc/BUILD
@@ -268,6 +268,7 @@ envoy_cmake(
     cache_entries = {
         "BUILD_CURL_EXE": "off",
         "BUILD_TESTING": "off",
+        "BUILD_LIBCURL_DOCS": "off",
         "BUILD_SHARED_LIBS": "off",
         "CURL_HIDDEN_SYMBOLS": "off",
         "CURL_USE_LIBSSH2": "off",

